client/package.json

```diff
@@ -209,9 +209,11 @@
 	"devDependencies": {
 		"@automattic/calypso-eslint-overrides": "workspace:^",
 		"@sentry/webpack-plugin": "^1.18.8",
+		"@testing-library/dom": "^8.13.0",
 		"@testing-library/jest-dom": "^5.16.2",
 		"@testing-library/react": "^12.1.3",
 		"@testing-library/react-hooks": "7.0.2",
+		"@testing-library/user-event": "13.5.0",
```

Comment:
Just flagging that `user-event@13` is no longer maintained — current stable version is `v14` (which is also the version that docs refer to)

You may want to update the version before merging this PR?

Reply:
@ciampo in my chats with @flootr I had the understanding that v14 requires React 18, which is why we're using v13.

Reply:
Interesting point — [it seems so](https://github.com/testing-library/user-event/blob/main/package.json#L60-L61). 

Although Gutenberg is using [version 14](https://github.com/WordPress/gutenberg/blob/trunk/package.json#L115) while still using [react 17](https://github.com/WordPress/gutenberg/blob/trunk/package.json#L208-L209)

Reply:
@tyxla @ciampo I pushed an attempt to using v14 via [fd13abf](https://github.com/Automattic/wp-calypso/pull/63757/commits/fd13abfbd5bd9b03ff95623c114e63149bdc431e). Just wanted to check whether that'd cause any problems.